posixprelude.bn

```diff
@@ -0,0 +1,29 @@
+;;;; posixprelude.bn -- Wrappers for POSIX functions   -*- bone -*-
+;;;; Copyright (C) 2016 Dov Murik
+;;;;
+;;;; Permission to use, copy, modify, and/or distribute this software for any
+;;;; purpose with or without fee is hereby granted, provided that the above
+;;;; copyright notice and this permission notice appear in all copies.
+;;;;
+;;;; THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+;;;; WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+;;;; MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+;;;; ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+;;;; WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+;;;; ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+;;;; OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+(defsub (gettimeofday)
+  "Return a list `(seconds microseconds)` of the time since epoch."
+  (aif (sys.gettimeofday?)
+       it
+    (err "Error calling POSIX gettimeofday, errno=" sys.errno)))
```

Comment:
`sys.errno` is a subroutine, so it should be `(sys.errno)`. It's a bit confusing because `errno` is a variable in C...